### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by lenemter**

### CODE REVIEW

File: po/extra/ru.po [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+msgid "Redesign app icons (big thanks to @Fatih20)"
+msgstr "Переработана иконка приложения (большое спасибо @Fatih20)"
+
+#: data/com.github.sgpthomas.hourglass.appdata.xml.in:165
+msgid "Update Brazilian Portuguese translation (by @gfavaro)"
+msgstr "Обновлён бразильский-португальский перевод (от @gfavaro)"
+
+#: data/com.github.sgpthomas.hourglass.appdata.xml.in:171
+msgid "Fixes:"
+msgstr "Исправления"
```
Comment (ryonakano) [inline po/extra/ru.po:295]: Is the lack of the trailing comma intentional here?

File: po/extra/ru.po [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+msgid "Add Japanese translation"
+msgstr "Добавлен японский перевод"
+
+#: data/com.github.sgpthomas.hourglass.appdata.xml.in:200
+msgid "Add Portuguese-Portugal translation (by @rottenpants466)"
+msgstr "Добавлен португальский перевод (от @rottenpants466)"
+
+#: data/com.github.sgpthomas.hourglass.appdata.xml.in:202
+msgid "Other updates:"
+msgstr "Другие обновления"
```
Comment (ryonakano) [inline po/extra/ru.po:370]: Is the lack of the trailing comma intentional here?

**Review by ryonakano — COMMENTED** [review]
Hi, thank you for your translation! Leaving a tiny confirmation, otherwise looks good :+1:

**Comment by lenemter** [comment]
Thanks for noticing. I fixed trailing colons

**Review by ryonakano — APPROVED** [review]
Nice, thank you for your quick fix! :tada:

**Pull request merged**